python/tk_multi_loader/model_publishhistory.py

```diff
@@ -82,6 +82,7 @@ def load_data(self, sg_data):
         filters = [ ["project", "is", sg_data["project"] ],
                     ["name", "is", sg_data["name"] ],
                     ["entity", "is", sg_data["entity"] ],
+                    ["sg_status_list", "not_in", ["omt", "ip"]],
```

Comment:
Extend the filters with the "publish_filters" settings from the env yml file.
You can get it that way: pub_filters = app.get_setting("publish_filters", []) (from model_latestpublish.py l181)